.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+script: rake bootstrap test:install-default && rake test:core test:plugins
```

Comment:
Well, in that case its super easy in travis, since it is by default clean. I just need to disable the caching directives I specified above. I guess what I'm saying is that I'm not sure if it's worth the slowdown, but I suppose that can be  resolved in a future PR. Should we also not cache `.gradle` then? I assume that's fine to cache. Right @colinsurprenant ?